.github/workflows/main.yml

```diff
@@ -156,6 +156,34 @@ jobs:
           retention-days: 7
           if-no-files-found: error
 
+  nb-javac-smokecheck:
+    name: "NetBeans on nb-javac"
+    # Run test job only when the 'nb-javac' label is added
```

Comment:
Is it wise to run the job only when a special label is added? We wouldn't catch #5748 so quickly without running this job.